CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.3.2 TBD
```

Comment:
could you parenthesize the TBD please? i think it's required for the tooling to automagically fill in the date :)